### PR TITLE
keepcache: Add test for remove command scenario (RhBug:2237883)

### PR DESCRIPTION
**Pull request by jan-kolarik**
We don't want to remove package cache when just removing packages.

Requires: https://github.com/rpm-software-management/dnf5/pull/885.

### CODE REVIEW

File: dnf-behave-tests/dnf/keepcache.feature [hunk 1]
```diff
@@ -79,3 +79,28 @@ Scenario: Reseting keepcache does not remove previously kept packages from cache
...
+
+
+@bz2237883
+Scenario: Cached packages are not removed after remove command with keepcache set to false
+  Given I use repository "scriptlets"
+    And I successfully execute dnf with args "install labirinto"
+   Then Transaction is following
+        | Action        | Package                                   |
+        | install       | labirinto-0:1.0-1.fc29.x86_64             |
+   When I execute dnf with args "install Package-pre-fail"
```
Comment (m-blaha) [inline dnf-behave-tests/dnf/keepcache.feature:91]: I might be missing something, but I think two install commands are over-complicating the test. I also do not like abusing `scriptlets` repository, I consider using --downloadonly switch clearer than relying on the transaction to fail due to a broken package. Would something like
```
Given I successfully execute dnf with args "install --downloadonly labirinto"
```
, then completely omitting "scriptlets" repository and Package-pre-fail installation, and then testing presence of `labirinto.rpm` in the cache work?

Comment (jan-kolarik) [reply dnf-behave-tests/dnf/keepcache.feature:91]: Hmm, you are right it is not intuitive. The labirinto package is not important here, I just want to be sure there is some package present which I use later for successful `remove` command usage. The point here is, that when packages are downloaded during **non-successful** transaction, they are kept after successfully running some `remove` command later. Which should match the use case from connected bz...

Comment (jan-kolarik) [reply dnf-behave-tests/dnf/keepcache.feature:91]: If it makes sense, I can try to add some comments there.

Comment (m-blaha) [reply dnf-behave-tests/dnf/keepcache.feature:91]: I think that having a failing install transaction is not important here (at least scenario name does not imply it). Important is to fill the cache with something and for this purpose will `install --downloadonly` serve well.

But I didn't realize you need to successfully remove something, so you need to also successfully install this something before. I mean - double install is needed, sorry for the confusion.

**Pull request merged**